### PR TITLE
column titlebar

### PR DESCRIPTION
**Pull request by kernelkind**
each column has title bar with:
- unique title specific to column type
- delete button on hover

on top of PR #340

drop 0a22210, 983640b, b1549a5 once https://github.com/damus-io/egui-nav/pull/2 is merged

### CODE REVIEW

**Review by jb55 — CHANGES_REQUESTED** [review]
A few issues:

# Bouncy Nav

almost there. one big issue is that the title is "bouncy" during nav. it seems to stack on top of each other. We should be doing that iOS does and put them at the same level, so that the nav does not move the view up and down. The view should not move vertically at all. The title nav should have a fixed size.

example:

IMG_2627
IMG_2625

# Simplify titles

Instead of "Your Notifications", let's just use "Notifications"

Instead of "Will's Home Timeline", lets just use "Contacts"

basically shorten them when it detects they are ours.

**Comment by kernelkind** [comment]
image

**Comment by kernelkind** [comment]
https://github.com/user-attachments/assets/a4387b38-6ec1-42b8-b866-ae42777eb7da

addressed comments. See recording for example

**Comment by jb55** [comment]
merged, thanks! I prefer the old arrow but not going to stop merging over it
